docs/jps_guide/titlepage.tex.in

```diff
@@ -0,0 +1,30 @@
+%% this file is generated automatically. All manual changes will be deleted.
+\newcommand{\HRule}{\rule{\linewidth}{0.5mm}}
+\begin{titlepage}
+\begin{center}
+
+\includegraphics[width=0.75\textwidth]{../images/jupedsim_small.png} \\ [1cm]
+
+\textsc{\LARGE Jülich Pedestrian Simulator} \\ [1.5cm]
+
+\textsc{\small Forschungszentrum Jülich GmbH} \\ [0.5cm]
+
+\HRule \\ [0.4cm]
+{ \huge \bfseries User's Guide Version @PROJECT_VERSION@} \\ [0.4cm]
+
+
+\HRule \\ [1.5cm]
+
+{\large Stand: \today}
```

Comment:
Below the date is set to `@GIT_DATE` and here `\today`  is used, leading to two different dates on the titlepage, which does not make sense in my opinion. Or is this independent from the info below?

Reply:
`\today` is the day of creation of the document

`git_date` is the date of the last commit. These are two different things.

Reply:
In my understanding there are no changes between `\today` and `git_date`. Hence even if the documentation is build `\today` the date should be `git_date` as this is the date of the latest change. But this more a question for my understanding than from blocking this PR.

Reply:
removed.